lib/cursor-position-view.coffee

```diff
@@ -49,13 +49,14 @@ class CursorPositionView extends HTMLElement
     atom.workspace.getActiveTextEditor()
 
   updatePosition: ->
-    if position = @getActiveTextEditor()?.getCursorBufferPosition()
-      @row = position.row + 1
-      @column = position.column + 1
-      @goToLineLink.textContent = @formatString.replace('%L', @row).replace('%C', @column)
-      @classList.remove('hide')
-    else
-      @goToLineLink.textContent = ''
-      @classList.add('hide')
+    atom.views.updateDocument () =>
```

Comment:
It looks like `atom.views.updateDocument` returns a `Disposable`. To be robust, this should keep a reference to that Disposable and call `.dispose` on it for the cases in which multiple `updatePosition` calls before a document update happens.